store/migration/10001__init_schema.sql

```diff
@@ -716,7 +716,8 @@ CREATE TABLE backup_setting (
     day_of_week INTEGER NOT NULL CHECK (
         -1 <= day_of_week
         AND day_of_week < 7
-    )
+    ),
+    url TEXT NOT NULL
```

Comment:
hook_url?  Only say "url" would be confusing as people may think it's the URL storing the backup.

Also please add a comment above as this is a not so obvious feature.